web/tests/functional/authentication/test_authentication.py

```diff
@@ -206,11 +206,24 @@ def try_login(self, provider, username, password):
             f"&code_challenge_method={code_challenge_method}" \
             f"&code_challenge={code_challenge}"
 
+        validate_result = auth_client.validateOAuthSession(state)
+        self.assertTrue(validate_result, "State in begigning and"
+                        " ending of OAuth is different")
+
         self.session_token = auth_client.performLogin(
             "oauth", provider + "@" + auth_string)
 
         return self.session_token
 
+    def test_oauth_token_session(self):
+        auth_client = env.setup_auth_client(
+            self._test_workspace, session_token='_PROHIBIT')
+
+        self.try_login("github", "admin_github", "admin")
```

Comment:
you could also create an assertion here for the return value of the try_login() function as it should be a valid codechecker session token